api/controllers/UserController.js

```diff
@@ -331,7 +331,7 @@ module.exports = {
           res.json('ok', 200);
         }, function(error) {
           console.log(error);
-          res.json({error: error}, 500);
+          res.json(error, 500);
         });
       });
     });
```

Comment:
It already does show body.error. The Promises return the object {error: 'message'}. This code was accidentally doubling it, causing the client to receive {error: {error: 'message'}}